snapcraft.yaml

```diff
@@ -941,6 +941,11 @@ parts:
       - --prefix=/usr
       - --with-builtin=pulse
     build-environment: *buildenv
+    override-build: |
+      set -eux
+      cd $CRAFT_PART_BUILD
```

Comment:
fwiw, this is the default work directory

Reply:
I will drop the line. For consistency the same should be done for the rest of snapcraft.yaml with a separate PR.